tests/js/lib/request.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var fluid = require("infusion");
+
+var kettle = require("kettle");
+kettle.loadTestingSupport();
+
+fluid.defaults("gpii.express.tests.request", {
```

Comment:
I am assuming you meant `gpii.tests.express`, which you used below and which is consistent with what I see in `infusion`.